lib/oppen.rb

```diff
@@ -94,72 +102,74 @@ def trim_trailing_whitespaces? = @trim_trailing_whitespaces
 
     def upsize_stack? = @upsize_stack
 
-    # Default config for Oppen usage
+    # Default config for Oppen usage.
+    #
     # @return [Config]
     def self.oppen
       new
     end
 
-    # Default config for Wadler usage
+    # Default config for Wadler usage.
+    #
     # @return [Config]
     def self.wadler(eager_print: true, trim_trailing_whitespaces: true, upsize_stack: true)
-      new(indent_anchor: IndentAnchor::ON_BEGIN, eager_print:, trim_trailing_whitespaces:, upsize_stack:)
+      new(eager_print:, indent_anchor: IndentAnchor::ON_BEGIN, trim_trailing_whitespaces:, upsize_stack:)
     end
   end
 
   # @param value [String]
-  # @param width [Integer] token width that defaults to value.length
+  # @param width [Integer] token width that defaults to value.length.
   #
-  # @return [Oppen::Token::String] a new String token
+  # @return [Token::String] a new String token.
   def self.string(value, width: value.length)
     Token::String.new(value, width:)
   end
 
   # @see Token::Whitespace
   #
-  # @return [Oppen::Token::Whitespace] a new Whitespace token.
+  # @return [Token::Whitespace] a new Whitespace token.
   def self.whitespace(value)
     Token::Whitespace.new(value, width: value.bytesize)
   end
 
-  # @param str [String]
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
-  # @param width [Integer] token width that defaults to str.length
+  # @param str               [String]
+  # @param line_continuation [String]  If a new line is needed display this string before the new line.
+  # @param offset            [Integer]
+  # @param width             [Integer] Token width that defaults to str.length.
   #
-  # @return [Oppen::Token::Break] a new Break token
+  # @return [Token::Break] a new Break token.
   def self.break(str = ' ', line_continuation: '', offset: 0, width: str.length)
     Token::Break.new(str, width:, line_continuation:, offset:)
   end
 
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
+  # @param line_continuation [String] If a new line is needed display this string before the new line.
+  # @param offset            [Integer]
   #
-  # @return [Oppen::Token::LineBreak] a new LineBreak token
+  # @return [Token::LineBreak] a new LineBreak token.
   def self.line_break(line_continuation: '', offset: 0)
     Token::LineBreak.new(line_continuation:, offset:)
   end
 
   # @param offset [Integer]
   #
-  # @return [Oppen::Token::Begin] a new consistent Begin token
+  # @return [Token::Begin] a new consistent Begin token.
```

Comment:
We need a word on what is consistent breaking, with an `@example`.

Also, we need a word on `offset`. Do not leave undocumented `@param`s. Even if they look "obvious".